src/helpers/messages.js

```diff
@@ -82,10 +80,11 @@ function getUnansweredItemsWarning(scope, runner, sync) {
             )}`;
         }
     } else if (scope === 'part') {
-        if (unansweredCount === 0) {
-            itemsCountMessage = __('You answered all %s question(s)', stats.questions.toString());
-        } else {
-            itemsCountMessage = __('You have %s unanswered question(s)', unansweredCount.toString());
+        itemsCountMessage = `<b>${__('You are about to submit this test part.')}</b><br><br>`
+        if (unansweredCount > 1) {
+            itemsCountMessage += __('There are %s unanswered questions in this part of the test', unansweredCount.toString());
+        } else if (unansweredCount === 1) {
+            itemsCountMessage += __('There is %s unanswered question in this part of the test', unansweredCount.toString());
         }
         if (flaggedCount) {
```

Comment:
```suggestion
        if (unansweredCount && flaggedCount) {
```

Reply:
maybe not show flagged if we don't have unanswered? In this case, we will not have sentence started with `and you flagged `